### ISSUE TITLE
Infinite loop on specific web pages

### ISSUE THREAD

**Issue by Gitoffthelawn** [claustromaniac/httpz#71]
I hope you are okay.  I don't think I've seen you around for over a year, and your GitHub public history shows nothing for 2020 and 2021.

On a few web pages, HTTPZ gets the browser (Firefox) stuck in an infinite loop.

An example is:
http://libqxt.org/

I thought perhaps this is another site to be added to https://github.com/claustromaniac/httpz/issues/55, but that issue report indicates an error message with buttons.  I don't receive any message or buttons... just an infinite loop.

I've experienced this on about a half-dozen sites out of hundreds.

**Comment by grahamperrin** [comment]
> I hope you are okay. 

Ditto. 

> http://libqxt.org

Please, do you have another example? https://downforeveryoneorjustme.com/libqxt.org it appears to be down.

**Comment by Gitoffthelawn** [comment]
> > I hope you are okay.
> 
> Ditto.

Yes. I wrote those words almost half a year ago, and no reply yet.  I hope @claustromaniac is okay.

> > http://libqxt.org
> 
> Please, do you have another example? https://downforeveryoneorjustme.com/libqxt.org it appears to be down.

Unfortunately, I do not have another example at this time.  I'm also not currently using the extension, although I might go back to using it someday.  So I probably won't have any other examples any time soon.

IIRC, the aforementioned site was up and running back in April, but I confirmed that it does appear to be down now.
